en/guide/wireshark.md

```diff
@@ -0,0 +1,111 @@
+# Parsing MAVLink in Wireshark
+
+## Generate MAVLink Lua Plugin for Wireshark
+
+First you will need to generate a *Wireshark* plugin that includes definitions for the MAVLink messages that you want it to handle.
+The MAVLink generator (**mavgen**) can build this plugin for a dialect in the same way as it builds MAVLink libraries for other programming languages.
+
+The steps are:
+1. [Install MAVLink](../getting_started/installation.md) (if you have not already done so).
+1. Build libraries for your target dialect, specifing `WLua` as the target language.
+   This process is described in the toic[Generate MAVLink Libraries](../getting_started/generate_libraries.md).
+
+   For example, to build the MAVLink 2 Wireshark plugin for [common.xml](../messages/common.md) you might use the following command:
+   ```bash
+   python3 -m pymavlink.tools.mavgen --lang=WLua --wire-protocol=2.0 --output=mavlink_2_common message_definitions/v1.0/common.xml
+   ```
+   The plugin would be created in the current directory as: **mavlink_2_common.lua**. 
+
+> **Note** You will need to regenerate and reimport the plugin (as shown below) if your dialect changes.
+
+## Import Lua Plugin into WireShark
+
+The import the plugin into *Wireshark*
+
+1. Copy the plugin file into the wireshare plugins directory.
+   - On Linux systems this might be: `~/.wireshark/plugins`
+     - If you are running *Wireshark* with sudo then do:
+       - `sudo su`
+       - Copy it into `/root/.local/lib/wireshark`
+  - On Windows this might be: `Program Files/Wireshark/plugins`.
+1. The last few lines of the plugin file specify the ports to be monitored.
+   By default these are 14550 and 14580, the ports for communication with a GCS and Offboard control.
+   Change these lines and save the file if your system uses different ports.
+1. Open *Wireshark* and follow the menu: **Help > About Wireshark > Plugins**
+
+   You should find the plugin in the list.
+   For example, with the plugin created in the previous section you would see `mavlink_2_common.lua` 
+
+## View Simulator Traffic
+
+You can view simulator traffic using directly using Wireshark.
+Simply select the interface for loopback traffic in the GUI.
+This will display all messages on the interface, with MAVLink message names next to each udp packet that contains a MAVLink message.
+
+You can filter on MAVLink properties to control which traffic is displayed.
+For example, the filter `mavlink_proto.sysid == 1` will filter on all messages that have a system id of 1, while `mavlink_proto.msgid==0 && mavlink_proto.compid == 1` will filter on HEARTBEAT messages (`msgid==0`) from an autopilot (`compid==1`).
+
+
+## Capture MAVLink Stream
```

Comment:
Because tcpdump can be run on a different device with console/ssh access (e.g. mission computer). This means you can tap into the traffic on the drone, between MAVLink components and to a ground station that is not your laptop.